packages/teleport-postprocessor-js-beautify-html/src/index.ts

```diff
@@ -0,0 +1,32 @@
+// @ts-ignore
+import { html as beautifyHTML } from 'js-beautify'
+
+import { PostProcessor, PrettierFormatOptions, FileType } from '@teleporthq/teleport-types'
+
+interface PostProcessorFactoryOptions {
+  fileType?: string
+  formatOptions?: PrettierFormatOptions
+}
+
+export const createPostProcessor = (options: PostProcessorFactoryOptions = {}) => {
+  const fileType = options.fileType || FileType.HTML
+
+  const processor: PostProcessor = (codeChunks) => {
+    if (codeChunks[fileType]) {
+      const result = beautifyHTML(codeChunks[fileType], {
+        wrap_line_length: 80,
+        end_with_newline: true,
+        preserve_newlines: true,
+      })
+      codeChunks[fileType] = result
+    } else {
+      console.warn('No code chunk of type HTML found, prettier-html did not perform any operation')
```

Comment:
make sure you rename the message here

Reply:
Yeah yeah 😅  this was for testing purpose. I just copied 😅